addons/miscFixes/XEH_crewTraining.sqf

```diff
@@ -0,0 +1,59 @@
+#include "script_component.hpp"
+
+// PabstMirror - Show reminder when non-crew enters vehicles seats that they might not be trained on
+LOG("Adding Crew Training Setting");
+
+[
+    QGVAR(crewTraining),
+    "CHECKBOX",
+    ["Show Crew Training Reminder", "Show reminder about crew training when entering a complex vehicle weapon"],
+    ["POTATO UI", "miscFixes"],
+    true, // default value
+    true, // isGlobal
+    {
+        params ["_enabled"];
+        TRACE_1("crewTraining setting updated",_enabled);
+
+        private _peID = missionNamespace getVariable [QGVAR(crewTraining_peID), -1];
+        if (_peID > -1) then { ["turret", _peID] call CBA_fnc_removePlayerEventHandler; };
+
+        if (!_enabled) exitWith {};
+
+        GVAR(crewTraining_peID) = ["turret", {
+            params ["_unit", "_turret"];
+            TRACE_2("turret EH",_unit,_turret);
+
+            if ((!alive _unit) || {_turret isEqualTo []}) exitWith { TRACE_1("dead or dismounted/cargo seat",_turret); };
+
+            private _unitType = toLower typeOf _unit;
+            if ((["pilot", "crew", "msv_cc", "msv_vic"] findIf {_x in _unitType}) > -1) exitWith { TRACE_1("trained",_unitType); };
```

Comment:
I want to say there's a third MSV for vehicles by the way, but maybe you've reduced slots since I looked at this repo.